Samples/iOS15-SwiftUI/iOS15-SwiftUI.xcconfig

```diff
@@ -22,7 +22,8 @@
 #include "../Shared/Config/Metal.xcconfig"
 
 PRODUCT_BUNDLE_IDENTIFIER = io.sentry.iOS15-SwiftUI
-INFOPLIST_FILE = iOS-SwiftUI/Info.plist
+INFOPLIST_FILE = iOS15-SwiftUI/Info.plist
```

Comment:
It seems to me like the Info.plist was misconfigured too

Reply:
Nice finds on these 👍🏻